### PR TITLE
Updated documentation

### LINKED ISSUE #1257: Update README.md with Security and Resources sections
It would be nice to add Security and Resources sections to the README like in [WCiOS README](https://github.com/woocommerce/woocommerce-ios/blob/develop/README.md) so that both docs have more consistent structure :)

### PR DESCRIPTION
**Pull request by AmandaRiu**
Updated project documentation:
- <a href="https://github.com/woocommerce/woocommerce-android/blob/amanda/1257_readme/README.md" rel="nofollow">README.md</a> : Revamped with a new layout and some new sections. Also fixes #1257 
- <a href="https://github.com/woocommerce/woocommerce-android/blob/amanda/1257_readme/README.md" rel="nofollow">LICENSE.md</a> : Converted to markdown.
- <a href="https://github.com/woocommerce/woocommerce-android/blob/amanda/1257_readme/CONTRIBUTING.md" rel="nofollow">CONTRIBUTING.md</a>: new!
- <a href="https://github.com/woocommerce/woocommerce-android/blob/amanda/1257_readme/CODE-OF-CONDUCT.md" rel="nofollow">CODE-OF-CONDUCT.md</a>: new!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+The core team monitors and reviews all pull requests. Depending on the changes, we will either approve them or close them with an explanation. We might also work with you to improve a pull request before approval.
+
+We do our best to respond quickly to all pull requests. If you don't get a response from us after a week, feel free to reach out to us via Slack.
+
+### Development Practices
+
+<!--TODO-->
+Have look at the [Coding Style Guide](https://github.com/woocommerce/woocommerce-android/wiki/Coding-Style) to learn how to format your code so it passes the convention and quality checks like the rest of the project. The [Wiki](https://github.com/woocommerce/woocommerce-android/wiki) contains all the development guides like [what a good pull request looks like](https://github.com/woocommerce/woocommerce-android/wiki/Pull-Request-Guidelines), and [how to use String and Drawable resources](https://github.com/woocommerce/woocommerce-android/wiki/Using-Android-Resources).
```
Comment (nbradbury) [inline CONTRIBUTING.md]: np: should be "Have **_a_** look."

**Comment by nbradbury** [comment]
Great job of this @AmandaRiu! I left a comment about a minor typo but other than that this is ready to :shipit:

File: CONTRIBUTING.md (hunk 1 above)
Comment (AmandaRiu) [reply CONTRIBUTING.md]: Nice catch! Fixed in 6e8fbea145ca098ec5f87d5e8cbf0fffc47640b6

**Comment by AmandaRiu** [comment]
@nbradbury or @anitaa1990  Ready for a second round!

**Pull request merged**